src/gatling/simulations/simulations/ET_Simulation.scala

```diff
@@ -66,16 +80,47 @@ class ET_Simulation extends Simulation {
         session
     }
 
+  //defines the Gatling simulation model, based on the inputs
+  def simulationProfile(simulationType: String, numberOfPerformanceTestUsers: Double, numberOfPipelineUsers: Double): Seq[OpenInjectionStep] = {
+    simulationType match {
+      case "perftest" =>
+        if (debugMode == "off") {
+          Seq(
+            rampUsers(numberOfPerformanceTestUsers.toInt) during (testDurationMins minutes)
```

Comment:
Is there a reason you've moved away from the open model simulation type here? We usually don't use rampUsers unless it's specifically needed for the test type.